docs/source/tutorials/notifications.rst

```diff
@@ -0,0 +1,217 @@
+Notifications
+=============
+
+If you read through :ref:`ref-chatops` section, you are familiar with notifications.
+Even without chatops, notifications can be used to post messages to external systems
+like chat clients, send emails etc. Notifications require an action that is registered with
+st2 (For example, ``slack`` pack contains a
+`post_message <https://github.com/StackStorm/st2contrib/blob/master/packs/slack/actions/post_message.yaml>`_ action.)
+and a notification rule to go with it. Notifications are implemented as triggers and rules and actions.
+A speical ``core.st2.notifytrigger`` is emitted by the system on completion of every action
+and a rule to match the trigger to a notify action results in notifications being sent out.
+
+How to setup a notification for a simple action?
+------------------------------------------------
+
+This is the easiest case. You can do this by specifying a ``notify`` section in the YAML meta
+while registering the action. For example:
+
+::
```

Comment:
I might be wrong, but IIRC, this syntax doesn't always auto-detect the language so it's better to use `.. sourcecode:: yaml`